Arrays/script.js

```diff
@@ -80,10 +80,28 @@ displayMovements(account1.movements);
 const calcDisplayBalance = function (movements) {
   const balance = movements.reduce((acc, mov) => acc + mov, 0);
   //console.log(balance);
-  labelBalance.textContent = `${balance} EUR`;
+  labelBalance.textContent = `${balance}€`;
 };
 calcDisplayBalance(account1.movements);
 
+const calcDisplaySummary = function (movements) {
+  const incomes = movements
+    .filter(mov => mov > 0)
+    .reduce((acc, mov) => acc + mov, 0);
+  labelSumIn.textContent = `${incomes}€`;
```

Comment:
I assume it's not part of this task, but generally, it's a good practice to have functions [pure](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976), e.g. include labelSumIn, labelSumOut,  labelSumInterest into the list of function parameters. Same for `calcDisplayBalance`.

Reply:
Ok, I will take note it...this code is part of study app written by Jonas